imports/plugins/core/accounts/server/no-meteor/getShopIdByDomain.test.js

```diff
@@ -1,26 +1,41 @@
 import getShopIdByDomain from "./getShopIdByDomain";
-import mockContext from "/imports/test-utils/helpers/mockContext";
```

Comment:
once again, I added the `describe` block which makes it appear as though the whole file is modified.
I added a `resetContext` helper method as well, to set strings back to default values to reset mock functions